edge/pkg/edged/edged.go

```diff
@@ -287,6 +288,23 @@ func (e *edged) Start(c *context.Context) {
 	e.context = c
 	e.metaClient = client.New(c)
 
+	autoRegister := config.CONFIG.GetConfigurationByKey("edged.register-node").(bool)
+	if autoRegister {
+		go func() {
+			// keep trying to register with apiserver every 10 seconds
+			for {
+				resource := fmt.Sprintf("%s/%s/%s", e.namespace, model.ResourceTypeNodeStatus, e.nodeName)
+				msg := message.BuildMsg(modules.MetaGroup, "", modules.EdgedModuleName, resource, model.InsertOperation, "")
+				_, err := e.context.SendSync("websocket", *msg, 60*time.Second)
```

Comment:
for `websocket` , Recommended to use constants 。 https://github.com/kubeedge/kubeedge/blob/8819f62b8b4db5f0f83528ad30a1747e9f933afb/edge/pkg/edgehub/module.go#L11